src/foam/mlang/mlang.js

```diff
@@ -3735,6 +3735,34 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.mlang.predicate',
+  name: 'IsSet',
+  extends: 'foam.mlang.predicate.Has',
+  implements: [ 'foam.core.Serializable' ],
+
+  documentation: 'Predicate which checks if given property is set(for FObjects only, for java objects we invoke HasPredicate)',
+
+  javaImports: [
+    'foam.core.FObject',
+    'foam.core.PropertyInfo'
+  ],
+
+  methods: [
+    {
+      name: 'f',
+      javaCode: `
```

Comment:
Do we need to do anything for JS, or do we just rely on Has for that?
Why do we need this, or where do we use it?